system-addon/content-src/components/TopSites/TopSites.jsx

```diff
@@ -46,10 +46,7 @@ class TopSite extends React.Component {
     let imageStyle;
     if (tippyTopIcon) {
       imageClassName = "tippy-top-icon";
-      imageStyle = {
-        backgroundColor: link.backgroundColor,
-        backgroundImage: `url(${tippyTopIcon})`
```

Comment:
Looks like tests want this. Could just put empty string (or leave it as `undefined`) and it should work?